### ISSUE TITLE
Not include some files

### ISSUE THREAD

**Issue by kingdev29** [RestKit/RestKit#2444]

**Comment by valeriomazzeo** [comment]
Which version of cocoapods are you using?

Restkit 0.27.0 is compatible with cocoapods 1.0.1 with use_frameworks! enabled

**Comment by kingdev29** [comment]
I'm using cocoapods 1.0.1 version.

**Comment by valeriomazzeo** [comment]
You shouldn't need to move or rename any file. If you were using AFNetworking directly into your project you may now want to add it to your podfile, regardless you are using RestKit or not as it's not an explicit dependency anymore.

As far as I know, it works if you create a new project and install RestKit. You may want to try that, it may be something going on in your specific project.

**Comment by kingdev29** [comment]
Let me try again as you said. Thanks
